apps/src/applab/DesignModeBox.jsx

```diff
@@ -59,6 +59,7 @@ module.exports = React.createClass({
           <div data-element-type="DROPDOWN" className="new-design-element">dropdown</div>
           <div data-element-type="IMAGE" className="new-design-element">image</div>
           <div data-element-type="CANVAS" className="new-design-element">canvas</div>
+          <div data-element-type="SCREEN" className="new-design-element">screen</div>
```

Comment:
Nit: I haven't seen the mocks, but it took me a second to figure out I had to drag a screen into the workspace to add it.  Might make more sense as an "Add Screen" or "+" button next to the screens dropdown.